pyproject.toml

```diff
@@ -1,3 +1,48 @@
+[project]
+name = "setuptools-rust"
+version = "1.6.0"
+description = "Setuptools Rust extension plugin"
+readme = "README.md"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
```

Comment:
I think it would be helpful if this used some machine readable form like an SPDX identifier? Or is the MIT license used customized?

Reply:
I think it's standard MIT; we have a trove classifier which documents it as MIT License.

I got the impression from https://peps.python.org/pep-0621/#license that I should not use an SPDX identifier here? @messense maybe you know what's the best way to set the license as MIT?

Reply:
I think the current version of this information is https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#license together with https://packaging.python.org/en/latest/specifications/core-metadata/#license which indeed does not suggest something like SPDX identifier.

My reading of the second link is actually that the `license` field should be left out completely if the license trove classifier fully captures the license.

Reply:
> My reading of the second link is actually that the license field should be left out completely if the license trove classifier fully captures the license.

Yes, I had thought the same though it wasn't perfectly clear. I've dropped `license` for now.